Content.IntegrationTests/Tests/Doors/AirlockTest.cs

```diff
@@ -1,170 +0,0 @@
-                Assert.That(doorComponent.State, Is.EqualTo(SharedDoorComponent.DoorState.Opening));
-            });
-
-            await server.WaitIdleAsync();
-
-            await WaitUntil(server, () => doorComponent.State == SharedDoorComponent.DoorState.Open);
-
-            Assert.That(doorComponent.State, Is.EqualTo(SharedDoorComponent.DoorState.Open));
-
-            server.Assert(() =>
-            {
-                doorComponent.Close();
-                Assert.That(doorComponent.State, Is.EqualTo(SharedDoorComponent.DoorState.Closing));
-            });
-
-            await WaitUntil(server, () => doorComponent.State == SharedDoorComponent.DoorState.Closed);
-
-            Assert.That(doorComponent.State, Is.EqualTo(SharedDoorComponent.DoorState.Closed));
-
-            server.Assert(() =>
-            {
-                Assert.DoesNotThrow(() =>
-                {
-                    airlock.Delete();
-                });
-            });
-
-            server.RunTicks(5);
-
-            await server.WaitIdleAsync();
-        }
-
-        [Test]
-        public async Task AirlockBlockTest()
-        {
-            var options = new ServerContentIntegrationOption
-            {
-                ExtraPrototypes = Prototypes
-            };
-            var server = StartServer(options);
-
-            await server.WaitIdleAsync();
-
-            var mapManager = server.ResolveDependency<IMapManager>();
-            var entityManager = server.ResolveDependency<IEntityManager>();
-
-            IPhysBody physBody = null;
-            IEntity physicsDummy = null;
-            IEntity airlock = null;
-            ServerDoorComponent doorComponent = null;
-
-            var physicsDummyStartingX = -1;
-
-            server.Assert(() =>
-            {
-                var mapId = new MapId(1);
-                mapManager.CreateNewMapEntity(mapId);
-
-                var humanCoordinates = new MapCoordinates((physicsDummyStartingX, 0), mapId);
-                physicsDummy = entityManager.SpawnEntity("PhysicsDummy", humanCoordinates);
-
-                airlock = entityManager.SpawnEntity("AirlockDummy", new MapCoordinates((0, 0), mapId));
-
-                Assert.True(physicsDummy.TryGetComponent(out physBody));
-
-                Assert.True(airlock.TryGetComponent(out doorComponent));
-                Assert.That(doorComponent.State, Is.EqualTo(SharedDoorComponent.DoorState.Closed));
-            });
-
-            await server.WaitIdleAsync();
-
-            // Push the human towards the airlock
-            Assert.That(physBody != null);
-            physBody.LinearVelocity = (0.5f, 0);
-
-            for (var i = 0; i < 240; i += 10)
-            {
-                // Keep the airlock awake so they collide
-                airlock.GetComponent<IPhysBody>().WakeBody();
-
-                // Ensure that it is still closed
-                Assert.That(doorComponent.State, Is.EqualTo(SharedDoorComponent.DoorState.Closed));
```

Comment:
Just remove this line and the test should pass. We can still use the test to make sure it stops entities when closed.